homeassistant/components/squeezebox/browse_media.py

```diff
@@ -80,7 +77,7 @@ async def build_item_response(player, payload):
     else:
         browse_id = None
 
-    result = await player.async_browse(
+    result = await entity.async_browse(
```

Comment:
This is weird. We pass the entity, call a method on the entity, and then process those results ?

Reply:
yah i didnt like this implementation either, any thoughts of how to better handle?

Reply:
whatever the entity calls, we call that here .

Reply:
Ok, fix this to pass the `_player` object of the entity so that `player.async_browse` can be called directly. The reason I had implemented it the other way was to limit the number of parameters since since we have to pass the entity anyway for the `get_browse_image_url` method, and `_player` is not exposed by the entity, but this is fine too.